### PR TITLE
VZ-7342: Prevent CLI upgrade if already at specified upgrade version

### PR DESCRIPTION
**Pull request by bobdonat**
This pull request is a change to the vz CLI upgrade command.  A check was added and an informational message is displayed when the installed version of vz is already at the specified upgrade version.  The upgrade does not proceed after the informational message.

### CODE REVIEW

**Review by mbenmclean — DISMISSED** [review]

**Review by stevshap — DISMISSED** [review]

File: tools/vz/cmd/upgrade/upgrade.go [hunk 1]
```diff
@@ -20,9 +20,10 @@ import (
 )
 
 const (
-	CommandName = "upgrade"
-	helpShort   = "Upgrade Verrazzano"
-	helpLong    = `Upgrade the Verrazzano Platform Operator to the specified version and update all of the currently installed components`
+	CommandName     = "upgrade"
+	helpShort       = "Upgrade Verrazzano"
+	helpLong        = `Upgrade the Verrazzano Platform Operator to the specified version and update all of the currently installed components`
+	alreadyUpgraded = "Verrazzano has already been upgraded to version %s\n"
```
Comment (pfmackin) [inline tools/vz/cmd/upgrade/upgrade.go:26]: What if user tries to upgrade to a version just installed?  Better to say...
"Verrazzano is already at version xyz"

Comment (bobdonat) [reply tools/vz/cmd/upgrade/upgrade.go:26]: I originally had this "Verrazzano is already at the specified upgrade version of %s\n".
I can change it back

Comment (mbenmclean) [reply tools/vz/cmd/upgrade/upgrade.go:26]: I prefer Verrazzano is already at the specified upgrade version of %s\n

Comment (bobdonat) [reply tools/vz/cmd/upgrade/upgrade.go]: I changed to "Verrazzano is already at the specified upgrade version of %s\n"

Comment (bobdonat) [reply tools/vz/cmd/upgrade/upgrade.go]: I can change if necessary

Comment (pfmackin) [reply tools/vz/cmd/upgrade/upgrade.go]: I guess that is ok

**Review by pfmackin — APPROVED** [review]

**Pull request merged**